apps/io-profile/src/main.ts

```diff
@@ -0,0 +1,248 @@
+import { Context } from "@azure/functions";
+import express from "express";
```

Comment:
Remove this import and any other unused ones.

Reply:
thank you, done in https://github.com/pagopa/io-auth-n-identity-domain/pull/256/commits/d3f64f655db0a101c94c85b50bd4ade63d19e896